src/electron/states/directive.state.ts

```diff
@@ -38,6 +44,11 @@ export class DirectiveState extends State {
     if (s) {
       if (s instanceof DirectiveSymbol || s instanceof ComponentSymbol) {
         return getDirectiveMetadata(s);
+      } else if (s instanceof InjectableSymbol) {
+        return getInjectableMetadata(s);
+      // TODO: Implement it, when Provider is gonna be provided by ngast
```

Comment:
We'll not get there. We have a hybrid injector now, mostly reflective so this information is not available at build time.

Feel free to remove the comments.

Reply:
Currently we get **s** as instance of **Provider** and it causes an error as it's handled as getElementMetadata and doesn't contain required properties. At least we need to handle such case somehow.

Reply:
Yes, feel free to return `null` because in ngast we will not be able to accomplish it.

I meant, removing the comments with the implementation and maybe adding a comment that this is not supported by the parsing library.

Reply:
Yeah, right.. I missed that there is no TemplateNode on directive/component state. It's only on template state.